CONTRIBUTING.md

```diff
@@ -4,13 +4,45 @@
 
 Please note we have a code of conduct, please follow it in all your interactions with the project.
 
-## Pull Request Process
+# Pull Request Process
+
+## Steps for the PR author
 
 1. Ensure that tests pass and code is lint free. You can run `yarn test` and `yarn lint` locally to check.
-2. Update the README.md if any changes invalidate its current content.
+2. Update the README.md if any changes invalidate or extend its current content.
 3. Include tests for any new functionality.
 4. Reference relevant issues in your PR comment.
 
+Unless your PR is ready for immediate review and merging, please mark it as 'draft' (or simply do not open a PR yet).
+
```

Comment:
We should consider adding policy around code owner responding to an opened PR within 2 days, especially when its from an outside contributor.

Reply:
Agree. We need a way to deal with draft PRs that just sit as drafts forever.

Reply:
I'm up for that. Is 2 _business days_ reasonable?

Reply:
> Agree. We need a way to deal with draft PRs that just sit as drafts forever.

@smartcontracts I think that in that case my preference would be the "(simply do not open a PR yet)".

IMO the only good reason for a draft is to get feedback on a WIP. Once the feedback is provided, maybe we should just close it then, along with a warm invitation to reopen when it's ready for final review?

Reply:
We could use a cron job based github action to close stale PRs - https://github.com/marketplace/actions/close-stale-issues
I kind of like having open draft PRs as a way to know what people are working on - if there was a better way to discover that information, I'd be open to it.

>  Is 2 business days reasonable?

Yes

Reply:
I created a ticket for that cron job:
https://github.com/ethereum-optimism/roadmap/issues/863